src/pyflex/window_selector.py

```diff
@@ -184,6 +184,21 @@ def default(self, obj):
             except TypeError:
                 filename.write(j.encode())
 
+    def write_text(self, filename):
+        """
+        Write plain text file for the window output
+        """
+        sta = self.observed.stats.station
+        nw = self.observed.stats.network
+        comp = self.observed.stats.channel
+        loc = self.observed.stats.location
+        f = open(filename, 'w')
```

Comment:
Please use a `with` statement to assure the file is closed in all circumstances.